src/rviz/mesh_loader.cpp

```diff
@@ -447,6 +465,7 @@ void loadMaterials(const std::string& resource_path,
 
     Ogre::Technique* tech = mat->getTechnique(0);
     Ogre::Pass* pass = tech->getPass(0);
+    pass->setVertexColourTracking(Ogre::TVC_DIFFUSE);
```

Comment:
this is impacting other meshes, e.g. `rosrun rviz mesh_marker_test` and you'll see some of the meshes flicker because of this change. I don't have a suggestion yet for a fix